src/common/store/accounts.slice.ts

```diff
@@ -39,19 +42,23 @@ const slice = createSlice({
       state.isFetching = false;
     },
     removeAccount(state, action: PayloadAction<IAccount>) {
-      // This reducer can be called when an account does not exist with this UUID, hence why this reducer is written differently that the others
+      // This reducer can be called when an account does not exist with this UUID, hence why this
+      // reducer is written differently that the others
       return { ...state, accounts: state.accounts.filter((a) => a.uuid !== action.payload.uuid) };
     },
     updateAccount(state, action: PayloadAction<IAccount>) {
       const idx = state.accounts.findIndex((a) => a.uuid === action.payload.uuid);
       state.accounts[idx] = action.payload;
     },
+    setAccountsToAdd(state, action: PayloadAction<SerializedWalletWithAddress[]>) {
+      state.accountsToAdd = action.payload;
```

Comment:
Syncing this leaks private keys to the main process. That seems like a bad idea 😄